UT4MasterServer/Services/DatabaseContext.cs

```diff
@@ -13,5 +13,13 @@ public DatabaseContext(IOptions<DatabaseSettings> settings)
 	{
 		client = new MongoClient(settings.Value.ConnectionString);
 		Database = client.GetDatabase(settings.Value.DatabaseName);
+
+		var statisticsCollection = Database.GetCollection<Statistic>("statistics");
+		var statisticsIndexes = new List<CreateIndexModel<Statistic>>()
+		{
+			new CreateIndexModel<Statistic>(Builders<Statistic>.IndexKeys.Ascending(indexKey => indexKey.AccountId)),
+			new CreateIndexModel<Statistic>(Builders<Statistic>.IndexKeys.Ascending(indexKey => indexKey.CreatedAt))
+		};
+		statisticsCollection.Indexes.CreateMany(statisticsIndexes);
```

Comment:
I'm not familiar with indexes... is it intentional to create them upon EVERY access to database? new instance of DatabaseContext is created on each request. what does this exactly do? i think this would be better suited to be placed in StatisticsService.

Reply:
The command is checking if the index with same definition and name already exists, so they only created once, but checked every time.
I will see to put this to another place, and run it only once when application starts.

Reply:
I already do something like this in CloudStorageService